tests/Filter.Tests/Generic/IEnumerable`1ExtensionsTests.cs

```diff
@@ -331,42 +396,6 @@ public void Should_filter_when_property_types_match_as_range_ushort()
                 }
             }
 
-            [Fact]
```

Comment:
The issue with them being there, is that our test runner was counting them as duplicate tests on each run. So each test would be run for each subclass. 

> 3 classes * 3 tests = 9 tests

Is that what we actually want?